src/VehicleSelect/VehicleSelect.tsx

```diff
@@ -11,6 +11,7 @@ function uniqueSortedArray(values: string[]) {
 // component to select a vehicle
 function VehicleSelect({
   flexDirection = "column",
+  limitTags = 1,
```

Comment:
Did you mean to update `VehicleSelect`? It seems from the PR description that you were working on the `VehicleSelector` component?

Reply:
Some tests were failing  for VehicleSelect without limitTag fixed that issue and removed changes from VehicleSelect component